### ISSUE TITLE
Build a 'finish' account creation page

### ISSUE THREAD

**Issue by briri** [CDLUC3/dmsp_frontend_prototype#14]
Build out a page that we can direct new users to when they first create an account and we do not get all of their info.

For example if the user authenticates in Cognito with a SAML (or OpenId) based system, and we do not get all of the info we want from that system, they should be redirected to a page asking them for that info.

We want to ensure that we have:
- The correct email address (required)
- The correct affiliation (required)
- The first and last name (required)
- ORCID
- alternate/backup email

Requires **dmsp_rails_prototype** issue [#10](https://github.com/CDLUC3/dmsp_rails_prototype/issues/10)

**Comment by bofstein** [comment]
Should show any information we do have for them but let them edit it, and show the missing fields to fill out

![Image](https://github.com/user-attachments/assets/161ef360-411d-468b-810c-e3f8bb2f850b)

**Comment by bofstein** [comment]
Likely we'll add an SSO redirect to an intermediate page that checks if they have missing information and sends them to an Update form if they're missing information and the dashboard (or wherever we want to send them) if their account is complete.

**Comment by bofstein** [comment]
Do we do this now?  The goal of allowing SSO is that they don't need to create an account.  I would assume we get the required info from the provider and then we wouldn't need this.  We might _want_ them to e.g. provide an ORCID, but making a page like this would be like mandating it, and that would go against the goal of using SSO to avoid account sign up

**Comment by briri** [comment]
Yes, we do this currently and absolutely need this. The only attributes an SSO is "required" to provide are the user's email address and their unique id (aka EPPN)  which within the SSO system. 

Sometimes we get a `Name` field like `Jane Doe` or `Doe, Jane` or `J. Doe`. Other times we get a separate first and last name, other times we get nothing.

Often times, the email address isn't the same as the one the SSO sends us. For example the user might provide `jane.doe@example.com`, get sent to the SSO login, they login and the SSO system sends use `jdoe@example.edu` in the email field. The user entered `jane.doe@example.com` though and either prefers using that or doesn't realize its an alias so we should allow them to use it.

We would show the affiliation/org name in the browser but not allow the user to change this since it is connected to the specific SSO system. 

We would not show them their EPPN as it typically has no value to the user and is only used by us behind the scenes to match what the SSO sends back on subsequent logins to a user in our system. These are sometimes the same as the email address but are typically something like `abcd1234@example.com`. 

We want the user to provide their first and last names, ORCID (maybe?), and confirm the email address that the SSO sent us.
